src/components/ExternalButtonLink.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button, ButtonProps } from "@chakra-ui/react"
+import { ExternalLinkHref } from "./Navbar/types"
+import { FC } from "react"
+
+const ExternalButtonLink: FC<ButtonProps & { href: ExternalLinkHref }> = ({
+  href,
+  ...props
+}) => {
+  return (
+    <Button
+      as="a"
+      href={href}
```

Comment:
Maybe we should force the `a` and `href` prop by:
```
    <Button
       rel="noopener noreferrer"
       target="_blank"
       {...props}
       as="a"
       href={href}
```

ofc non-blocking comment.